assembl/models/preferences.py

```diff
@@ -463,6 +475,50 @@ def get_preference_data_list(self):
             "default": True
         },
 
+        # What are (ordered) identifiers of columns in multi-column views?
+        {
+            "id": "default_column_identifiers",
+            "name": _("Ids of columns in column view"),
+            "value_type": "list_of_string",
+            "description": _(
+                "What are (ordered) identifiers of columns in multi-column views?"),
+            "allow_user_override": None,
+            "modification_permission": P_ADMIN_DISC,
+            # "frontend_validator_function": func_name...?,
+            # "backend_validator_function": func_name...?,
+            "default": ["positive", "negative"],
+        },
+
+        # What are default theme colors of columns in multi-column view
+        {
+            "id": "default_column_colors",
+            "name": _("Default colors of columns in column view"),
+            "value_type": "strdict_of_string",
+            "description": _(
+                "What are (default) theme colors of columns in multi-column views?"),
+            "allow_user_override": None,
+            "modification_permission": P_ADMIN_DISC,
+            # "frontend_validator_function": func_name...?,
+            # "backend_validator_function": func_name...?,
+            "default": {"positive": "green", "negative": "red"},
```

Comment:
Many considerations:
Column colours are set for an individual column, hence for a given idea, within a given discussion, which has a given theme. So you can set any colour for any column, already taking the theme in the account.
What we have here in the preferences is the default colors when you create a new column set. It's easy to change it for that column afterwards.
It's a discussion preference, so you could absolutely change those defaults for a given discussion to fit the discussion's theme, so new column sets for that discussion follow the theme. I think this is fine.